app/src/main/java/com/codesoom/assignment/application/UserService.java

```diff
@@ -0,0 +1,96 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.UserNotFoundException;
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.domain.UserRepository;
+import com.codesoom.assignment.dto.UserData;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import com.github.dozermapper.core.Mapper;
+
+import java.util.List;
+
+/**
+ * 유저를 관리합니다.
+ */
+@Service
+public class UserService {
+
+    private final UserRepository userRepository;
+    private final Mapper mapper;
+
+    @Autowired
+    public UserService(UserRepository userRepository, Mapper dozerMapper) {
+        this.userRepository = userRepository;
+        this.mapper = dozerMapper;
+    }
```

Comment:
여기에서 `@Autowired`가 없어도 Spring은 `UserRepository`와 `Mapper`를 잘 주입해 줄 겁니다. 왜 그런지 이유를 조사해 보세요.
```suggestion
    public UserService(UserRepository userRepository, Mapper dozerMapper) {
        this.userRepository = userRepository;
        this.mapper = dozerMapper;
    }
```